packages/sync-actions/src/tax-categories.js

```diff
@@ -1,34 +1,49 @@
 /* @flow */
 import flatten from 'lodash.flatten'
-import type { SyncAction, ActionGroup } from 'types/sdk'
+import type {
+  SyncAction,
+  ActionGroup,
+  UpdateAction,
+  SyncActionConfig,
+} from 'types/sdk'
 import createBuildActions from './utils/create-build-actions'
 import createMapActionGroup from './utils/create-map-action-group'
 import * as taxCategoriesActions from './tax-categories-actions'
 import * as diffpatcher from './utils/diffpatcher'
 
 export const actionGroups = ['base', 'rates']
 
-function createTaxCategoriesMapActions(mapActionGroup) {
-  return function doMapActions(diff, newObj, oldObj /* , options */) {
+function createTaxCategoriesMapActions(
+  mapActionGroup: (
+    type: string,
+    fn: () => Array<UpdateAction>
+  ) => Array<UpdateAction>,
+  config: SyncActionConfig
+): (diff: Object, newObj: Object, oldObj: Object) => Array<UpdateAction> {
+  return function doMapActions(
+    diff: Object,
+    newObj: Object,
+    oldObj: Object
+  ): Array<UpdateAction> {
     const allActions = []
-
     allActions.push(
-      mapActionGroup('base', () =>
-        taxCategoriesActions.actionsMapBase(diff, oldObj, newObj)
+      mapActionGroup('base', (): Array<UpdateAction> =>
+        taxCategoriesActions.actionsMapBase(diff, oldObj, newObj, config)
       )
     )
-
     allActions.push(
-      mapActionGroup('rates', () =>
+      mapActionGroup('rates', (): Array<UpdateAction> =>
         taxCategoriesActions.actionsMapRates(diff, oldObj, newObj)
       )
     )
-
     return flatten(allActions)
   }
 }
 
-export default (config: Array<ActionGroup>): SyncAction => {
+export default (
+  actionGroupConfig: Array<ActionGroup>,
+  config: SyncActionConfig
```

Comment:
Maybe rename to `syncActionConfig` it's a bit confusing when you read on and see the createTaxCategoriesMapAction receive "just config".